keanu-project/src/main/java/io/improbable/keanu/util/Py4jUtils.java

```diff
@@ -0,0 +1,34 @@
+package io.improbable.keanu.util;
+
+import lombok.experimental.UtilityClass;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+@UtilityClass
+public class Py4jUtils {
+    private static final int JAVA_DOUBLE_SIZE = 8;
+    private static final int JAVA_INT_SIZE = 4;
```

Comment:
you can use `Double.SIZE` and `Integer.SIZE` - just need to divide by 8 since they're measured in bits.